config/jobs/kubernetes-sigs/sig-windows/release-master-windows.yaml

```diff
@@ -230,18 +230,17 @@ periodics:
     testgrid-dashboards: sig-windows-master-release, sig-windows-signal
     testgrid-tab-name: capz-windows-containerd-master-serial-slow-hpa
 - name: ci-kubernetes-e2e-capz-master-windows-2022
+  cluster: k8s-infra-prow-build
   interval: 3h
   decorate: true
   decoration_config:
     timeout: 4h
   labels:
     preset-dind-enabled: "true"
     preset-kind-volume-mounts: "true"
-    preset-azure-cred-wi: "true"
-    preset-azure-anonymous-pull: "true" # Sets REGISTRY which is needed when building CCM/CNM images
+    preset-azure-community: "true"
     preset-capz-windows-common: "true"
     preset-capz-containerd-1-7-latest: "true"
-    preset-windows-private-registry-cred: "true"
```

Comment:
Just a note this will break the test `Kubernetes e2e suite.[It] [sig-node] Container Runtime blackbox test when running a container with a new image should be able to pull from private registry with secret [NodeConformance]` we are tracking fixing this in https://github.com/kubernetes-sigs/windows-testing/issues/446